WordPress/src/main/java/org/wordpress/android/ui/domains/management/DomainStatusRow.kt

```diff
@@ -18,70 +18,107 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
 import androidx.compose.ui.unit.dp
+import org.wordpress.android.fluxc.network.rest.wpcom.site.DomainStatus
+import org.wordpress.android.fluxc.network.rest.wpcom.site.StatusType
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import java.time.format.FormatStyle
 
 @Composable
 fun StatusRow(
-    status: DomainStatus,
-    expiry: LocalDate,
+    status: DomainStatus?,
+    expiry: LocalDate?,
     modifier: Modifier = Modifier,
 ) {
     Row(
         modifier = modifier,
         verticalAlignment = Alignment.CenterVertically,
     ) {
-        Box(
-            modifier = Modifier
-                .size(8.dp)
-                .clip(CircleShape)
-                .background(status.statusUiState.indicatorColor),
-        )
-        Spacer(modifier = Modifier.width(8.dp))
-        Text(
-            text = status.statusUiState.text,
-            color = status.statusUiState.textColor,
-            style = if (status.statusUiState.isBold) {
-                LocalTextStyle.current.copy(fontWeight = FontWeight.Bold)
-            } else {
-                LocalTextStyle.current
-            },
-        )
-        Text(
-            text = "Expires ${expiry.mediumFormat}",
-            textAlign = TextAlign.End,
-            color = MaterialTheme.colorScheme.secondary,
-            modifier = Modifier.weight(1f),
-        )
+        status?.also { status ->
+            BulletPoint(color = status.indicatorColor)
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = status.status ?: "Unknown",
+                color = status.textColor,
+                style = if (status.isBold) {
+                    LocalTextStyle.current.copy(fontWeight = FontWeight.Bold)
+                } else {
+                    LocalTextStyle.current
+                },
+            )
+        } ?: run {
+            PendingGhostStrip(80.dp)
+        }
+        Spacer(modifier = Modifier.weight(1f))
+        expiry?.also { expiry ->
+            Text(
+                text = "Expires ${expiry.mediumFormat}",
```

Comment:
And same here. If it's not temporary, I'd put it to `strings.xml`

Reply:
Good idea. Resolved w/ https://github.com/wordpress-mobile/WordPress-Android/pull/19269/commits/f3a00c199ee8f07c9f6ff764faecfbbacb41d79c#diff-6bbab92723bb83a1300e86d496d62b9475a6a1e8b5a5feca27721edfbb09943fR2702.